valet/predicates.go

```diff
@@ -51,6 +51,7 @@ var markdownRegex = regexp.MustCompile(fmt.Sprintf(".*[.]%s$", MarkdownSuffix))
 var pdfRegex = regexp.MustCompile(fmt.Sprintf(".*[.]%s$", PDFSuffix))
 var csvRegex = regexp.MustCompile(fmt.Sprintf(".*[.]%s$", CSVSuffix))
 var gzipRegex = regexp.MustCompile(fmt.Sprintf(".*[.]%s$", GzipSuffix))
+var reportRegex = regexp.MustCompile(fmt.Sprintf("report.*[.]%s$", MarkdownSuffix))
```

Comment:
Surprised to see it expressed as `[.]` and not `\\.`.

Reply:
Oh, that's force of habit from Perl. Perlcritic forces use of the former and has shouted at me so many times that now I have Stockholm Syndrome.